packages/patterns/package.json

```diff
@@ -0,0 +1,77 @@
+{
+  "name": "@endo/patterns",
+  "version": "0.1.0",
+  "description": "Description forthcoming.",
+  "keywords": [],
+  "author": "Endo contributors",
+  "license": "Apache-2.0",
+  "homepage": "https://github.com/endojs/endo/tree/master/packages/patterns#readme",
```

Comment:
the fragment only works for projects, not files in the tree.

should be: https://github.com/endojs/endo/blob/master/packages/patterns/README.md

Incidentally, it's empty and needs contents.

Reply:
See https://github.com/endojs/endo/pull/1480

(Separately, still needs contents)

Reply:
The empty READMEs for pass-style, patterns, and exo will be minimally fixed by https://github.com/endojs/endo/pull/1480 with initial text that's much better than nothing. A first step.